include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -598,7 +602,7 @@ struct __parallel_copy_if_static_single_group_submitter<_Size, _ElemsPerItem, _W
                     const ::std::uint16_t __subgroup_id = __subgroup.get_group_id();
                     const ::std::uint16_t __subgroup_size = __subgroup.get_local_linear_range();
 
-#if _ONEDPL_SYCL_SUB_GROUP_LOAD_STORE_PRESENT
+#if _ONEDPL_LIBSYCL_SUB_GROUP_LOAD_STORE_PRESENT
```

Comment:
1. This is a "logic mix" of preprocessor code and C++ code.... It looks a mess..
I would recommend "have a look" on C++ code which remains after the preprocessor keeps one branch "#if" and the other branch "#else" and make the code clean-up (here and everywhere in that PR).

2. BTW, `_ONEDPL_LIBSYCL_SUB_GROUP_LOAD_STORE_PRESENT` is defined as identical zero.
https://github.com/oneapi-src/oneDPL/pull/1979/files?diff=split&w=0#diff-521f7db06a55567c1a1dffa855dde585fb6bc5fbe8633d19b528356e3a501ea0R91

M.b. it makes sense to remove the branches `_ONEDPL_LIBSYCL_SUB_GROUP_LOAD_STORE_PRESENT` at all?